compiler-rt/test/profile/instrprof-write-buffer.c

```diff
@@ -0,0 +1,42 @@
+// UNSUPPORTED: target={{.*windows.*}}
+// This test is derived from "compiler-rt/test/profile/instrprof-write-buffer-internal.c",
+// and that test was disabled on Windows due to sanitizer-windows bot failures.
+// Doing the same here. See 2fcc3f4b18.
+//
+// RUN: rm -f %t.buf.profraw
```

Comment:
Add CHECK lines to show value profiles are generated.

Reply:
Added CHECK lines for llvm-profdata output, but is there any other way to show value profiles are generated ? llvm-profdata does fail without code changes.